JSQMessagesViewController/Controllers/JSQMessagesViewController.m

```diff
@@ -239,7 +239,7 @@ - (void)viewWillAppear:(BOOL)animated
 {
     [super viewWillAppear:animated];
     self.toolbarHeightConstraint.constant = self.inputToolbar.preferredDefaultHeight;
-    [self.view layoutIfNeeded];
+    [self.view setNeedsLayout];
```

Comment:
Hey @alattis, I was thinking whether it's actually better to remove this line because the system will call `viewDidLayoutSubviews` when it presents the `JSQMessagesViewController`.